Docs/API.md

```diff
@@ -840,6 +844,7 @@ __Parameters__
 | ``fileName``  | _string_  | File to upload |
 | ``contentType``  | _string_ | Content type of the file. Defaults to " |
 | ``metadata``  | _Dictionary<string,string>_ | Dictionary of meta data headers and their values.Defaults to null.|
+| ``sse``    | _ServerSideEncryption_ | server side encryption option | Optional parameter. Defaults to null |
```

Comment:
`server side encryption` - needs to be `Server-side encryption` for consistency

Reply:
done.